tests/test_clinvar_to_evidence_strings.py

```diff
@@ -1,21 +1,20 @@
 import os
 import unittest
 
+from eva_cttv_pipeline import consequence_type as CT
```

Comment:
Please be consistent when using aliases: in the main code `ct` is used, but in the tests it is called `CT`.